common/djangoapps/student/models.py

```diff
@@ -180,13 +178,6 @@ def anonymous_id_for_user(user, course_id, save='DEPRECATED'):
     # This part is for ability to get xblock instance in xblock_noauth handlers, where user is unauthenticated.
     assert user
 
-    if save != 'DEPRECATED':
```

Comment:
Am I right that there is no any test to check deprecated param?

Reply:
I've checked tests using the search, run the student module tests locally and hope to find out if any test will fail using the GH pipeline

Reply:
waiting for the pipeline to finish to ensure all  is ok

Reply:
Great, then waiting until tests passed and open PR to upstream